certification/internal/policy/container/has_unique_tag_test.go

```diff
@@ -61,6 +68,13 @@ var _ = Describe("UniqueTag", func() {
 				Expect(ok).To(BeFalse())
 			})
 		})
+		Context("When registry returns an empty tag list", func() {
+			It("should pass Validate", func() {
```

Comment:
To me, it would make sense for this test to have its own `BeforeEach` with the `mockRegistry` in it, then to add to the existing before each. But not sure what others think.

Reply:
I think I would wrap the existing tests in a Context()/Describe(), and add the new tests to their own, with their own BeforeEach.